frameworks/SapientML/setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Please update the script so you can install both from source (as `latest`) and from pypi (as `stable` or with a specified version). See for example GAMA's script https://github.com/openml/automlbenchmark/blob/master/frameworks/GAMA/setup.sh